internal/client/config.go

```diff
@@ -20,10 +21,23 @@ import (
 	"sigs.k8s.io/yaml"
 )
 
+var (
+	// TestRootDir is the root directory for test files.
+	testRootDir string
```

Comment:
do we need to make this a global or could it be done another way? When I make a new client could the ENV be consumed?

Reply:
I guess we could carry it in the struct like `baseDir`, yes. Would just need to do that in a constructor and was probably shying away from changing so many places where we're using complex literals to create config.

Reply:
This would be only an issue if we start running unit tests in parallels. May be we should document this somewhere? I am confident the race detector would catch it as soon as we enabled it though.